platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNodeTest.java

```diff
@@ -253,18 +254,19 @@ void detachTest() {
     @Test
     @DisplayName("Default flush threshold not zero")
     void defaultFlushThresholdTest() {
-        final VirtualMapSettings settings = VirtualMapSettingsFactory.get();
+        final VirtualMapConfig config = ConfigurationHolder.getConfigData(VirtualMapConfig.class);
```

Comment:
In a test it would make more sense to use `TestConfigBuilder` instead of `ConfigurationHolder`